src/api.js

```diff
@@ -71,7 +71,7 @@ const disableKeyboardTrigger = () => {
 
 const onKeyDown = (event) => {
   keyDownStack += event.key;
-  if (event.key === 'F12' || keyDownStack.toUpperCase() !== 'F12') {
+  if ((event.ctrl && event.key === 'F12') || keyDownStack.toUpperCase() !== 'CONTROLF12') {
```

Comment:
We can't trust the keyDownStack because of those Mac keyup events in Citrix. The old if statement had an "or" to make sure tapping the F12 key didn't trigger it so we need to leave that logic as-is so let's throw that in a variable, then we can throw the new trigger in another variable and check them both. 

Untested, I think this will do it: 
```js
  const legacyF12 = event.key !== 'F12' && keyDownStack.toUpperCase() === 'F12';
  const ctrlF12 = event.key === 'F12' && event.ctrlKey;
  if (!legacyF12 && !ctrlF12) {
    return;
  }
```